parts/common/components.json

```diff
@@ -910,8 +910,8 @@
             "versionsV2": [
               {
                 "renovateTag": "<DO_NOT_UPDATE>",
-                "latestVersion": "1.6.18",
-                "previousLatestVersion": "1.5.38"
+                "latestVersion": "1.6.21",
```

Comment:
It looks like we're going from a 1.6 and a 1.5 to two 1.6.x versions - is this correct or do we want a 1.5.x version on the VHD as well as a 1.6.x version?

Reply:
oooh...something is already wrong, we should have stayed within minor versions.

Reply:
I'm wondering how the previous version could have been 1.5.38 in the first place-- usually we keep the two latest 1.5.x versions and two latest 1.6.x versions since there is both an active 1.5 version in aks and a 1.6 version in aks at all times (based on what k8 version you create your cluster with)

Reply:
when the VHD gets full, we may drop teh previous versions...this is either a bad merge or someone saw a 1.5 and a 1.6 and thought the 1.5 was the previous version for the 1.6 and collapsed them

Reply:
> For windows we typically keep one patch version rather than two because we regularly run out of VHD space - the containers can be huge.

I've migrated our latest versions to the Windows HPC base image, so we should contribute less to this 🙂 1.6+ so once AKS 1.27 LTS ends in July we'll be done with the old fat images for CNS, CNI